src/entities/protocols/seaport.ts

```diff
@@ -161,7 +174,7 @@ export class SeaportTrade extends NFTTrade<SeaportData> {
 
   private commandMap(protocolAddress: string): CommandType {
     switch (protocolAddress.toLowerCase()) {
-      case '0x00000000000000ADc04C56Bf30aC9d3c0aAF14dC': // Seaport v1.5
+      case '0x00000000000000adc04c56bf30ac9d3c0aaf14dc': // Seaport v1.5
```

Comment:
Because people could send the address as checksummed, or as capitals, or as lowercase and all need to be accepted. So I do `protocolAddress.toLowerCase()` and then compare with the lowercase version